README.md

```diff
@@ -1,15 +1,173 @@
-# rungs
-> Native wrapper for gobstones/gs-weblang-core
+# gs-weblang-cli
+CLI for the Gobstones interpreter
 
-## Requirements
+## Install
 
-* node 4
-* [nexe](https://github.com/jaredallard/nexe): `npm install nexe -g` 
+```bash
+# requires node > 4
+sudo npm install -g gs-weblang-cli
+```
 
 ## Building wrapper
 
+The cli can be wrapped as an executable using [nexe](https://github.com/jaredallard/nexe): `npm install nexe -g`.
+
+```
+git clone https://github.com/gobstones/gs-weblang-cli gobs
+cd gobs
+nexe -i src/index.js -o gobs
+```
+
+## Usage
+
+```bash
+gobs --help
+```
+
+### Run
+
+#### passed
+
+```bash
+echo "program {\n Poner(Rojo)\n }" > /tmp/gobs.gbs
+gobs /tmp/gobs.gbs -f gbb
 ```
-git clone https://github.com/mumuki/rungs rungs
-cd rungs
-nexe  -i index.js -o rungs
+
+```json
+{
+  "status": "passed",
+  "result": {
+    "x": 0,
+    "y": 0,
+    "sizeX": 9,
+    "sizeY": 9,
+    "table": "GBB/1.0\r\nsize 9 9\r\ncell 0 0 Azul 0 Negro 0 Rojo 1 Verde 0\r\nhead 0 0\r\n"
+  }
+}
+```
+
+#### compilation_error
+
+```bash
+echo "programita {\n Poner(Rojo)\n }" > /tmp/gobs.gbs
+gobs /tmp/gobs.gbs
+```
+
+```json
+{
+  "status": "compilation_error",
+  "result": {
+    "on": {
+      "range": {
+        "start": {
+          "row": 0,
+          "column": 1
+        },
+        "end": {
+          "row": 0,
+          "column": 10
+        }
+      },
+      "value": "programita",
+      "arity": "name"
+    },
+    "message": "Se esperaba una definición de programa, función o procedimiento."
+  }
+}
+```
+
+#### runtime_error
+
+```bash
+echo "program {\n Ponerrrrrrr(Rojo)\n }" > /tmp/gobs.gbs
+gobs /tmp/gobs.gbs
+```
+
+```json
+{
+  "status": "runtime_error",
+  "result": {
+    "on": {
+      "token": {
+        "range": {
+          "start": {
+            "row": 1,
+            "column": 2
+          },
+          "end": {
+            "row": 1,
+            "column": 10
+          }
+        },
+        "value": "Ponerrrrr",
+        "arity": "name"
+      }
+    },
+    "message": "El procedimiento Ponerrrrr no se encuentra definido."
+  }
+}
+```
+
+#### all_is_broken_error
+
+This one **shouldn't** happen very often, but for now... :sweat_smile:
+
+```bash
+echo "" > /tmp/gobs.gbs
+gobs /tmp/gobs.gbs
+```
+
+```json
+{
+  "status": "all_is_broken_error",
+  "message": "Something has gone very wrong",
+  "detail": {},
+  "moreDetail": "Cannot read property 'range' of null"
+}
+```
+
+### Batch run
+
+`request.json`:
+```json
+[
+  {
+    "initialBoard": "GBB...",
+    "code": "program {\n ..."
+  },
+  {
+    "initialBoard": "GBB...",
+    "code": "program {\n ..."
+  },
+]
+```
+
+```bash
+gobs --batch request.json
+# returns an array with the responses
+```
+
+### Generate AST
+
+```bash
+echo "program {\n Poner(Azul)\n }" > /tmp/gobs.gbs
+gobs --ast /tmp/gobs.gbs
```

Comment:
I mean, I am not against having a file input option, but default should still be reading from `stdin`